### ISSUE TITLE
Perl CGI "startform" / "endform" are obsolete - DocDB not working at all.

### ISSUE THREAD

**Issue by pms967** [ericvaandering/DocDB#12]
Perl CGI "startform" and "endform" methods have been long deprecated:

 http://perldoc.perl.org/CGI.html#STARTING-AND-ENDING-A-FORM

...and no longer exists in recent version of Perl CGI module:
```
*** /var/log/apache2//error.log ***
[...] [cgi:error] [...] AH01215: Can't locate object method "startform" via package "CGI" at DocumentDatabase line 132.: [...]
```
As a consequence, DocDB does not work on modern Linux distributions (such as Ubuntu 16.04 ad Debian Stretch. Expect the same to be true for new RHEL and derived distributions as well).

**Comment by pms967** [comment]
As a quick workaround, you may try to add this code at the end of DocDBGlobals.pm :

```
# fix for obsolete startform/endform
package CGI;
our %EXPORT_TAGS;
$EXPORT_TAGS{':standard'} = [ @{$EXPORT_TAGS{':standard'}},
                                  'startform', 'endform' ];
sub startform {
        &start_form;
    }
sub endform {
        &end_form;
    }

```

**Comment by ericvaandering** [comment]
Fixed in #14 which is working its way to release
